pkg/services/apiserver/builder/admission_test.go

```diff
@@ -76,6 +92,61 @@ func TestBuilderAdmission_Validate(t *testing.T) {
 	}
 }
 
+func TestNewAdmissionFromBuilders(t *testing.T) {
```

Comment:
i added this one to ensure that the pointers to the validation functions in the `range` loop weren't changing during the next iteration of the loop